maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/HardshipOrchestrationService.java

```diff
@@ -120,7 +122,10 @@ public ApplicationDTO update(WorkflowRequest request) {
 
     private void validate(WorkflowRequest request, Action action, RepOrderDTO repOrderDTO) {
         validationService.validate(request, repOrderDTO);
-        validationService.isUserActionValid(hardshipMapper.getUserActionDTO(request, action));
+
+        UserActionDTO userActionDTO = hardshipMapper.getUserActionDTO(request, action);
+        UserSummaryDTO userSummaryDTO = userActionDTO.getUsername() != null ? maatCourtDataService.getUserSummary(userActionDTO.getUsername()) : null;
```

Comment:
We never used to try to get the username from the now _UserActionDTO_ until after we had checked the following in the `isUserActionValid` method:

```
if (request.getAction() == null && request.getNewWorkReason() == null && request.getSessionId() == null) {
            throw new IllegalArgumentException(ACTION_NEW_WORK_REASON_AND_SESSION_DOES_NOT_EXIST);
        }
```

Given that we now have to try and access this DTO I figured it's probably best to check it's not null, and if it is, I'm assuming that we should be passing a `null` value through for the _UserSummaryDTO_ but this is where I'd like to get other folks' opinions.

Reply:
Looks reasonable, although I think a regular if statement would be more readable here.

Reply:
Can we meet halfway with something like:

`Optional.ofNullable(userActionDTO.getUsername()).map(maatCourtDataService::getUserSummary).orElse(null);`

Agree that an if statement would improve readability though at the expense of bloating the code. It's a shame there is no null safe operator in Java yet (such as the talked about Elvis operator).